lib/src/badge_position.dart

```diff
@@ -5,41 +5,41 @@ import 'package:badges/badges.dart';
 /// This contain a custom position for the [Badge]
 class BadgePosition {
   /// This is the distance to the top side of a parent widget.
-  final double top;
+  final double? top;
 
   /// This is the distance to the end side of a parent widget.
-  final double end;
+  final double? end;
 
   /// This is the distance to the start side of a parent widget.
-  final double start;
+  final double? start;
 
   /// This is the distance to the bottom side of a parent widget.
-  final double bottom;
+  final double? bottom;
 
   /// This creates a new instance of this widget
   const BadgePosition({this.top, this.end, this.bottom, this.start});
 
   /// Factory method that creates a new instance of this widget
   /// according to [top] and [start]
-  factory BadgePosition.topStart({double top, double start}) {
+  factory BadgePosition.topStart({double? top, double? start}) {
     return BadgePosition(top: top ?? -5, start: start ?? -10);
```

Comment:
instead of making the parameters nullable, I would set these with the default values below

same for `topEnd`, `bottomEnd` and `bottomStart`

Reply:
Done.